js/core/component.js

```diff
@@ -160,7 +122,7 @@ var Component = Class.inherit({
             this._setOptionsByReference();
             this._setDeprecatedOptions();
             this._options = this._getDefaultOptions();
-            this._optionManager = new OptionManager(
+            this._optionManager = new Options(
```

Comment:
Why we use `this._optionManager` here for `Options` and the same name `this._optionManager` for `OptionManager` instance inside the Options class?

Reply:
The same (see above):
In next PRs we remove all optionManager from components